heat/core/random.py

```diff
@@ -528,9 +528,13 @@ def randint(
     # sanitize shape
     if size is None:
         size = (1,)
-    shape = tuple(int(ele) for ele in size)
-    if not all(ele > 0 for ele in shape):
-        raise ValueError("negative dimensions are not allowed")
+    try:
+        shape = tuple(int(ele) for ele in size)
+    except TypeError:
+        shape = (int(size),)
+    else:
+        if not all(ele > 0 for ele in shape):
+            raise ValueError("negative dimensions are not allowed")
 
```

Comment:
I was thinking of addressing this like we do for `shape` in `ht.reshape()`:
´´´python
def reshape(a: DNDarray, *shape: Union[int, Tuple[int, ...]], **kwargs) -> DNDarray:
   ...
´´´

Default `size` should be `0`, not `None`. Here a numpy example:
```bash
>>> d = np.random.randint(0,50)
>>> d
1
>>> d = np.random.randint(0,50, 1)
>>> d
array([0])
>>> e = ht.random.randint(0,50)
>>> e
DNDarray([9], dtype=ht.int32, device=cpu:0, split=None)  # ndim=1 but should be 0!
```